Bottom: ade60b2ff4ded75f0d728a6be915e57014e039ab
Top:    2ce87ba65ad7b890efca46ec8b9d56231af9f7fc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-04-21 02:08:30 +0100

Fix borast not to build for non-GL enabled builds


---

diff --git a/src/Makefile.am b/src/Makefile.am
index bd12421..73cdc6f 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -38,22 +38,6 @@ PCB_SRCS = \
 	box.h \
 	buffer.c \
 	buffer.h \
-	borast/borast-minimal.h \
-	borast/borastint-minimal.h \
-	borast/borast-bentley-ottmann.c \
-	borast/borast-combsort-private.h \
-	borast/borast-compiler-private.h \
-	borast/borast-fixed-private.h \
-	borast/borast-fixed-type-private.h \
-	borast/borast-freelist.c \
-	borast/borast-freelist-private.h \
-	borast/borast-malloc-private.h \
-	borast/borast-traps.c \
-	borast/borast-traps-private.h \
-	borast/borast-types-private.h \
-	borast/borast-wideint.c \
-	borast/borast-wideint-private.h \
-	borast/borast-wideint-type-private.h \
 	change.c \
 	change.h \
 	clip.c \
@@ -197,8 +181,23 @@ GL_SRCS= \
 	hid/common/hidgl.c \
 	hid/common/hidgl.h \
 	hid/common/trackball.c \
-	hid/common/trackball.h
-
+	hid/common/trackball.h \
+	borast/borast-minimal.h \
+	borast/borastint-minimal.h \
+	borast/borast-bentley-ottmann.c \
+	borast/borast-combsort-private.h \
+	borast/borast-compiler-private.h \
+	borast/borast-fixed-private.h \
+	borast/borast-fixed-type-private.h \
+	borast/borast-freelist.c \
+	borast/borast-freelist-private.h \
+	borast/borast-malloc-private.h \
+	borast/borast-traps.c \
+	borast/borast-traps-private.h \
+	borast/borast-types-private.h \
+	borast/borast-wideint.c \
+	borast/borast-wideint-private.h \
+	borast/borast-wideint-type-private.h
 
 BUILT_SOURCES = \
 	core_lists.h \
